liquidctl/driver/kraken3.py

```diff
@@ -23,6 +23,12 @@
 _WRITE_LENGTH = 64
 _MAX_READ_ATTEMPTS = 12
 
+_STATUS_TEMPERATURE = 'Liquid temperature'
+_STATUS_PUMP_SPEED = 'Pump speed'
+_STATUS_PUMP_DUTY = 'Pump duty'
```

Comment:
I'm not sure, since doing the translation here would break API callers on non-English locales.

Maybe the CLI would be a better place for it: since at that point we have the keys _and_ the values, I think it'd still be doable, even considering languages that require a bit more care (e.g. for plurals).